blockdag/process_test.go

```diff
@@ -103,6 +104,7 @@ func TestProcessDelayedBlocks(t *testing.T) {
 
 	blockDelay := time.Duration(dag1.Params.TimestampDeviationTolerance)*dag1.Params.TargetTimePerBlock + 5*time.Second
 	delayedBlock.Header.Timestamp = initialTime.Add(blockDelay)
+	delayedBlock.Header.Nonce = 1
```

Comment:
Why?

Reply:
Even if you pass BFNoPOW to ProcessBlock, PoW is always checked in the un-delaying process.

Reply:
Open a ticket to fix it